Dockerfile

```diff
@@ -176,8 +176,22 @@ COPY vendor /go/src/github.com/docker/docker/vendor
 RUN go install -v github.com/cpuguy83/go-md2man \
 	github.com/BurntSushi/toml/cmd/tomlv
 
+# Build the tool for embedding resources in Windows binaries
+ENV RSRC_COMMIT e48dbf1b7fc464a9e85fcec450dddf80816b76e0
+RUN set -x \
+    && git clone https://github.com/akavel/rsrc.git /go/src/github.com/akavel/rsrc \
+    && (cd /go/src/github.com/akavel/rsrc \
```

Comment:
Immediately after the cd is where it does the checkout and subsequent build (not install). Have fixed the tabs though - darn editors! Will force push updates shortly.